code/modules/antagonists/eldritch_cult/knowledge/flesh_lore.dm

```diff
@@ -25,6 +25,8 @@
 	var/mob/living/carbon/human/humie = locate() in atoms
 	if(!humie)
 		return
+	if(humie.stat != DEAD)
+		return
 
 	humie?.grab_ghost()
```

Comment:
also i don't wanna be that guy but now just looking at this do we even need the ? if we return early on !humie

Reply:
you are correct

Reply:
you could get rid of the ? but not the if(!humie) check